src/get-started/text-fields/TextInput.js

```diff
@@ -4,11 +4,11 @@ import './TextInput.css';
 export default (props) => {
     return(
         <span className='input-label'>
+            <div className='name-text'>{`${props.name[0].toUpperCase()}${props.name.substring(1)}${props.info}`}</div>
```

Comment:
Could you create a nested function inside of this component to return that string? it's a bit hard to read.